assignments/mini-libc/libc/Makefile

```diff
@@ -0,0 +1,34 @@
+NASM = nasm
+CPPFLAGS = -nostdinc -Iinclude
+CFLAGS = -Wall -Wextra -fno-PIC -fno-stack-protector -fno-builtin
```

Comment:
Add right below this line debugging flags:
`# CFLAGS += -g -O0 -std=c99 # For debugging.`